sql/char_fishing.sql

```diff
@@ -0,0 +1,22 @@
+-- --------------------------------------------------------
+-- Add history for player fishing data
```

Comment:
This comment isn't like the other ones you've added, make it more like those - or remove it. It's pretty obvious these days that all of these files are defining tables.